src/main/java/org/jgroups/protocols/kubernetes/KUBE_PING.java

```diff
@@ -206,6 +218,33 @@ public void findMembers(List<Address> members, boolean initial_discovery, Respon
                 list.stream().filter(phys_addr -> !cluster_members.contains(phys_addr)).forEach(cluster_members::add);
         }
 
+        if (split_clusters_during_rolling_update) {
+            if(physical_addr != null) {
+                String senderIp = ((IpAddress)physical_addr).getIpAddress().getHostAddress();
```

Comment:
@belaban Could you please have a look at this? Can I safely cast to `IpAddress`?

Reply:
No, this should be find as I don't have other impls of PhysicalAddress